react-frontend/src/ProfilePopup.js

```diff
@@ -0,0 +1,82 @@
+import { useState } from "react";
+import Modal from "react-bootstrap/Modal";
+import "bootstrap/dist/css/bootstrap.min.css";
+
+function ProfilePopup(props) {
+  const [oldPassword, setOldPassword] = useState(null);
+  const [newPassword, setNewPassword] = useState(null);
+  const [confirmedNewPassword, setConfirmedNewPassword] = useState(null);
+
+  function handleChange(event) {
+    const { name, value } = event.target;
+    if (name === "oldPassword") setOldPassword(value);
+    else if (name === "newPassword") setNewPassword(value);
+    else setConfirmedNewPassword(value);
+  }
+
+  return (
+    <Modal
+      show={true}
+      onHide={() => {
+        props.setShow(false);
+      }}
+    >
+      <Modal.Header>
+        <button
+          className="close"
+          data-dismiss="modal"
+          aria-hidden="true"
+          onClick={() => {
+            props.setShow(false);
+          }}
+        >
+          ×
+        </button>
+      </Modal.Header>
+      <Modal.Body>
+        <div>
+          <form>
+            <label htmlFor="oldPassword">Old Password</label>
+            <input
+              type="text"
```

Comment:
Can you make the input type "password" instead of "text" (Similar to what was done in signup and login)

Reply:
Thank you, I was looking for what would cause that effect